packages/patternfly-4/react-core/src/components/Select/SelectToggle.js

```diff
@@ -0,0 +1,156 @@
+import React, { Component } from 'react';
+import styles from '@patternfly/patternfly/components/Select/select.css';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+import { CaretDownIcon } from '@patternfly/react-icons';
+import { KeyTypes } from './selectConstants';
+
+const propTypes = {
+  /** HTML ID of dropdown toggle */
+  id: PropTypes.string.isRequired,
+  /** Anything which can be rendered as dropdown toggle */
+  children: PropTypes.node,
+  /** Classes applied to root element of dropdown toggle */
+  className: PropTypes.string,
+  /** Flag to indicate if select is expanded */
+  isExpanded: PropTypes.bool,
+  /** Callback called when toggle is clicked */
+  onToggle: PropTypes.func,
+  /** Callback for toggle open on keyboard entry */
+  onEnter: PropTypes.func,
```

Comment:
onEnter, onClose, type all missing default props.  Are they also needed for the d.ts file?

Reply:
Will add